demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -276,6 +286,7 @@ def run_dev_packages(self, parallel: int, no_flake8: bool, no_xsoar_linter: bool
 
         with concurrent.futures.ThreadPoolExecutor(max_workers=parallel) as executor:
             return_exit_code: int = 0
+            return_warning_code: int = 0
```

Comment:
Can't we just print if there are warnings?  All other changes related to the report looks good, just need to verify some edge case, with warnings/without warnings/ warnings and errors/Empty (Which I believe you did).